Rock/Reporting/DataFilter/Group/GroupTypeDataViewFilter.cs

```diff
@@ -0,0 +1,322 @@
+        /// <value>
+        ///     The client format script.
+        /// </value>
+        public override string GetClientFormatSelection( Type entityType )
+        {
+            return @"
+function() {
+  var dataViewName = $('.rock-drop-down-list,select:first', $content).find(':selected').text();
+  var GroupType = $('.rock-drop-down-list,select:last', $content).find(':selected').text();
+  var result = 'GroupType';
+  if (GroupType) {
+     result = result + ' type ""' + GroupType + "";
+  }  
+  result = result + ' is in filter: ' + dataViewName;
+  return result;
+}
+";
+        }
+
+        /// <summary>
+        /// Provides a user-friendly description of the specified filter values.
+        /// </summary>
+        /// <param name="entityType">The System Type of the entity to which the filter will be applied.</param>
+        /// <param name="selection">A formatted string representing the filter settings.</param>
+        /// <returns>
+        /// A string containing the user-friendly description of the settings.
+        /// </returns>
+        public override string FormatSelection( Type entityType, string selection )
+        {
+            var settings = new FilterSettings( selection );
+
+            string result = "Connected to Group Type";
+
+            if (!settings.IsValid)
+            {
+                return result;
+            }
+
+            using (var context = new RockContext())
+            {
+                var dataView = new DataViewService( context ).Get( settings.DataViewGuid.GetValueOrDefault() );
+
+                string GroupTypeName = null;
+
+                if (settings.GroupTypeGuid.HasValue)
+                {
+                    GroupTypeName = DefinedValueCache.Read( settings.GroupTypeGuid.Value, context ).Value;
+                }
+
+                result = string.Format( "Group Type {0} is in filter: {1}",
+                                        ( GroupTypeName != null ? "type \"" + GroupTypeName + "\"" : string.Empty ),
+                                        ( dataView != null ? dataView.ToString() : string.Empty ) );
+            }
+
+            return result;
+        }
+
+        private const string _CtlDataView = "ddlDataView";
+
+        /// <summary>
+        /// Creates the child controls.
+        /// </summary>
+        /// <param name="entityType">Type of the entity.</param>
+        /// <param name="parentControl">The parent control.</param>
+        /// <returns></returns>
+        public override Control[] CreateChildControls( Type entityType, FilterField filterControl )
+        {
+            // Define Control: History Data View Picker
+            var ddlDataView = new DataViewPicker();
+            ddlDataView.ID = filterControl.GetChildControlInstanceName( _CtlDataView );
+            ddlDataView.Label = "Connected to Group Types";
+            ddlDataView.Help = "A Data View that provides the list of GroupTypes to which the Group may be connected.";
```

Comment:
Since this new filter is pretty straight-forward and is just filtering on matching GroupTypes ([line 312](https://github.com/SparkDevNetwork/Rock/pull/1933/files#diff-79adee19a7703fcf4547dee95db0fe82R312)), would you be open to changing (or allowing us to change):

1.  the `Label` to "Has a Group Type in this Data View" (similar to the [Group Member Data View filter's label](https://github.com/SparkDevNetwork/Rock/blob/develop/Rock/Reporting/DataFilter/Person/GroupMemberDataViewFilter.cs#L242) ).
2. the `Help` to say "A Data View that provides the set of Group Types to match."